### PR TITLE
linter: improved error output in PHPDoc

### PR DESCRIPTION
**Pull request by i582**
The line on which the error is now is the context line.
Also, the underline now highlights the desired part, rather than
the entire line (for functions, for example).

Before:
```
WARNING phpdocLint: @param for non-existing argument $package at testdata/options-resolver/OptionsResolver.php:428
    public function setDeprecated(string $option/*, string $package, string $version*/): self
                    ^^^^^^^^^^^^^
```

After:

```
WARNING phpdocLint: @param for non-existing argument $package at testdata/options-resolver/OptionsResolver.php:424
     * @param string          $package The name of the composer package that is triggering the deprecation
                              ^^^^^^^^
```

### CODE REVIEW

File: src/linter/phpdoc_util.go [hunk 1]
```diff
@@ -12,27 +12,61 @@ import (
 	"github.com/VKCOM/noverify/src/types"
 )
 
-type phpdocErrors struct {
-	phpdocLint []string
-	phpdocType []string
+type PHPDocPlace struct {
```
Comment (quasilyte) [inline src/linter/phpdoc_util.go:15]: Maybe we can move it to `phpdoc` package?
Along with related functions like PHPDocLine.

Also, Place should probably be called Location.

File: src/linter/phpdoc_util.go [hunk 2]
```diff
@@ -12,27 +12,61 @@ import (
...
+	Node  ir.Node
+	Line  int
+	Field int
+	All   bool
```
Comment (quasilyte) [inline src/linter/phpdoc_util.go:19]: What does "All" mean here?

Comment (i582) [reply src/linter/phpdoc_util.go]: Renamed with `WholeLine`.

File: src/linter/phpdoc_util.go (hunk 1 above)
Comment (i582) [reply src/linter/phpdoc_util.go]: Done.

**Pull request merged**